databricks/koalas/series.py

```diff
@@ -355,6 +355,31 @@ def index(self):
             raise KeyError('Currently supported only when the Column has a single index.')
         return self._kdf.index
 
+    @property
+    def is_unique(self):
+        """
+        Return boolean if values in the object are unique
+
+        .. note:: if the values contain `None`, the result is `False`.
+
+        Returns
+        -------
+        is_unique : boolean
+
+        >>> ks.Series([1, 2, 3]).is_unique
+        True
+        >>> ks.Series([1, 2, 2]).is_unique
+        False
+        >>> ks.Series([1, 2, 3, None]).is_unique
```

Comment:
this should be true right?

Reply:
Actually, it gives `False` in my local:

```
>>> pd.Series([1, 2, 3, None]).is_unique
False
```

Maybe version difference. let me check